tests/page/page-mouse.spec.ts

```diff
@@ -264,14 +264,18 @@ it('should tween mouse movement', async ({ page, browserName, isAndroid }) => {
   ]);
 });
 
-it('should always round down', async ({ page }) => {
+it('should always round down', async ({ page, browserName }) => {
   await page.evaluate(() => {
     document.addEventListener('mousedown', event => {
       window['result'] = [event.clientX, event.clientY];
     });
   });
   await page.mouse.click(50.1, 50.9);
-  expect(await page.evaluate('result')).toEqual([50, 50]);
+  // In _bidi* the rounding is browser-specific.
+  if (browserName === '_bidiFirefox')
+    expect(await page.evaluate('result')).toEqual([50, 51]);
+  else
+    expect(await page.evaluate('result')).toEqual([50, 50]);
```

Comment:
I wonder why it is rounded down on Chrome while we are rounding to the next closest integer. In this case we should really have the same result. Let me check internally if that is a bug on our side or not.

Reply:
No idea, I guess it's just an implementation detail, but let met know if you'd like to align the implementations.

Reply:
It's actually an issue on our side. So I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1960531. 

Based on that please remove the extra if condition so that Firefox will pass the test once we floor correctly as well. This was most likely an oversight.

Reply:
Ok, thank for confirming. In that case we can just keep the test as is.